src/NHibernate.Test/NHSpecificTest/NH3386/Fixture.cs

```diff
@@ -60,7 +60,7 @@ public void ShouldSupportNonRuntimeExtensionWithoutEntityReference()
 
 	public static class SqlServerFunction
 	{
-		[LinqExtensionMethod]
+		[LinqExtensionMethod(DBOnly = true)]
```

Comment:
No requirement here. Their working code will still work without it. Their failing code could start to work if they add it.